### ISSUE TITLE
Save States not working

### ISSUE THREAD

**Issue by apanloco** [Voljega/amiga4recalbox#15]
Great work! So excited to play some Amiga on my recalbox.

I have noticed that save states are not working. Any idea why?

I press _START_ to bring up the preferences.
Then i select save states, slot 0, and Save.
It doesn't complain, the preferences closes, but It's not possible to Load any save states because there is none.

**Comment by Voljega** [comment]
I don't thing the save states are working in amiberry for now : https://github.com/midwan/amiberry/issues/49

at least with whdl it might work with adf but I would have to try

**Comment by Voljega** [comment]
tried it this morning and couldn't find where the corresponding file is saved ? gonna ask on amiberry's forum

**Comment by Voljega** [comment]
yep it's working but there is a bug in my solution : https://github.com/midwan/amiberry/issues/192

it will be corrected in official integrated version

**Comment by apanloco** [comment]
Thanks!

**Comment by apanloco** [comment]
Where can I follow the development of the official integrated version?

**Comment by Voljega** [comment]
Hello, well these days we mostly talk about it on Recalbox forum, next
version should be done late this year or early next year

Le sam. 1 sept. 2018 à 20:26, Daniel Åkerud <notifications@github.com> a
écrit :
